content/en/tracing/setup/java.md

```diff
@@ -37,14 +37,14 @@ Finally, add the following JVM argument when starting your application in your I
 
 **Note**:
 
-* The `-javaagent` needs to be run before the `-jar` file, adding it as a JVM option, not as an application argument. For more information, see the [Oracle documentation][16].
+* The `-javaagent` needs to be run before the `-jar` file, adding it as a JVM option, not as an application argument. For more information, see the [Oracle documentation][4].
```

Comment:
The footnote renumbering is from a post-commit hook.
Now actually part of my semantic change.